tests/Backend/Snowflake/CloneIntoWorkspaceTest.php

```diff
@@ -112,6 +112,19 @@ public function testClone($aliasNestingLevel): void
         $workspaceTableData = $backend->fetchAll('languagesDetails');
         $this->assertCount(5, $workspaceTableData);
 
+        // try to import table from workspace
+        // table languagesDetails has _timestamp column
+        // but import still works
+        $this->_client->writeTableAsyncDirect(
```

Comment:
Jsem na masteru, ale test mi prochází. Není to proto, že jsou tam vyjmenované sloupce?

Reply:
mě ten test teda neprojde
```
Keboola\StorageApi\ClientException : Invalid columns: _timestamp: Only alphanumeric characters and underscores are allowed in column name. Underscore is not allowed on the beginning.
 /code/src/Keboola/StorageApi/Client.php:2832
 /code/src/Keboola/StorageApi/Client.php:2821
 /code/src/Keboola/StorageApi/Client.php:2788
 /code/src/Keboola/StorageApi/Client.php:2671
 /code/src/Keboola/StorageApi/Client.php:1155
 /code/tests/Backend/Snowflake/CloneIntoWorkspaceTest.php:121
```

Reply:
🤷  zkusím to znova

Reply:
Už mi taky failuje 🤦